newrelic_lambda_cli/cli/layers.py

```diff
@@ -87,6 +87,12 @@ def register(group):
     show_default=True,
     help="Enable/disable the New Relic Lambda Extension",
 )
+@click.option(
+    "--enable-extension-function-logs/--disable-extension-function-logs",
```

Comment:
Could this just be `enable-extension-logs`?

Reply:
I think it's important to maintain the distinction between extensions logs (logs generated by the extension, which are not captured) and function logs (which are).

Reply:
And of course, we already have another way of handling regular function logs, so the extension bit needs to be there. It's definitely wordy though.